app/controllers/submissions_controller.rb

```diff
@@ -275,7 +275,17 @@ def score
 
     # Submit the score
     if @submission.users.map {|user| lti_outcome_service?(@submission.exercise, user, @submission.study_group_id) }.any?
-      transmit_lti_score
+      url = finalize_submission_path(@submission)
+      submit_info = transmit_lti_score
+      submit_info&.[](:status)&.each do |s|
```

Comment:
Why are you iterating over the `status` here? If I read the code correctly, the status never contains more than one element, does it?

For reference: In `transmit_lti_score`, you're creating a new, empty array for the status and are later on pushing symbols to it. However, the pushing of a status is performed outside a loop and conditional as part of an `if-else` statement, avoiding two status information to be pushed. Further, I was not able to reproduce any code path for this case.